cmd/agent/main.go

```diff
@@ -91,6 +91,26 @@ See https://docs.c3os.io/after_install/upgrades/#manual for documentation.
 		},
 	},
 
+	{
+		Name:      "boot",
+		Usage:     "boot",
```

Comment:
A spare though: we could have a generic 'notify' that accepts an event as argument. In that case we could then stream down all the events coming from elemental too

Reply:
I like that option, the only reason this is in the agent is to leverage the manager.